xc/common/cmake/arch_define.cmake

```diff
@@ -62,6 +62,7 @@ function(ADD_XC_ARCH_DEFINE)
       --router_heap bucket \
       --clock_modeling route \
       --place_delay_model delta_override \
+      ----place_effort_scaling device_circuit \
```

Comment:
Let's wait for another a PR to enable this.

Reply:
Sure, commit removed.